### ISSUE TITLE
SA1516 (ElementsMustBeSeparatedByBlankLine) should allow property accessors to omit blank line when they are all one line

### ISSUE THREAD

**Issue by AArnott** [DotNetAnalyzers/StyleCopAnalyzers#693]
The following should be an allowed property definition by SA1516. The original StyleCop allowed it. 

```csharp
public string DisplayName
{
	get { return this.displayName; }
	set { this.displayName = value; }
}
```

However, as soon as *one* of the accessors became multi-line, they both had to be multi-line and a blank line is required between them.

**Comment by sharwell** [comment]
@AArnott another pro-tip - click edit to see how I updated the formatting so syntax highlighting for C# appears. :+1:

**Comment by AArnott** [comment]
Thanks

**Comment by sharwell** [comment]
I initially assigned this to @pdelvo since he originally implemented the rule. Let me know if you wanted to go ahead and fix it.

**Comment by pdelvo** [comment]
grabbed
